### PR TITLE
Adding `mutate_features` and `mutate_samples`

### PR DESCRIPTION
**Pull request by mikelove**
Addresses

https://github.com/tidyomics/tidyomics/issues/17

### CODE REVIEW

**Comment by mikelove** [comment]
I lumped in the `cli` change and i think fixed the workflow issue with this last commit

**Review by stemangiola — CHANGES_REQUESTED** [review]
For the bioc workflow, you are right that is finicky. I have updated to `rworkflow` and in the master it passes 2/3 OS. 

Do you mind reverting the changes to check-bioc (which I deleted from master). So we see where is failing and maybe I can help solving the errors/warnings.

Thanks!

**Comment by mikelove** [comment]
Ok I reverted and made a clean commit just adding `cli` to the print method

File: R/print_method.R [hunk 1]
```diff
@@ -32,12 +33,12 @@ tbl_format_header.tidySummarizedExperiment <- function(x, setup, ...) {
...
             # Add further info single-cell
-            append(sprintf(
-                "\033[90m Features=%s | Samples=%s | Assays=%s\033[39m",
+            append( cli::col_br_black( sprintf(
+                " Features=%s | Samples=%s | Assays=",
                 number_of_features,
                 number_of_samples,
                 assay_names %>% paste(collapse=", ")
-            ), after = 1)
+            )), after = 1)
```
Comment (stemangiola) [inline R/print_method.R:41]: Great, this would be amazing in another PR. This is quite fundamental. And I want to replicate this in all tidy transcriptomics.  But I don't want to have you too much work

**Comment by mikelove** [comment]
Yeah, this should resolve the printing but wherever you've been using ANSI color codes.

My next aim is some strategy for summarization avoiding nesting the SE itself.

**Comment by stemangiola** [comment]
> Yeah, this should resolve the printing bug wherever you've been using ANSI color codes.
> 
> My next aim is some strategy for summarization avoiding nesting the SE itself.

we have `group_split`, but the `group_by` and `with_groups` challenge remained pending. It required some creativity I believe to find a general solution.

**Comment by stemangiola** [comment]
And, @mikelove  welcome to the tidySE team! ;)
